### ISSUE TITLE
TLS provider has changed and it is not compatible with the current code

### ISSUE THREAD

**Issue by joscabmar** [kube-champ/terraform-k8s-sealed-secrets-bootstrap#1]
Seems like the Terraform TLS provider has changed and `key_algorithm` argument is no longer needed in `tls_self_signed_cert`:
```
╷
│ Error: Invalid Configuration for Read-Only Attribute
│
│   with tls_self_signed_cert.this,
│   on main.tf line 11, in resource "tls_self_signed_cert" "this":
│   11:   key_algorithm   = "RSA"
│
│ Cannot set value for this attribute as the provider has marked it as
│ read-only. Remove the configuration line setting the value.
│
│ Refer to the provider documentation or contact the provider developers for
│ additional information about configurable and read-only attributes that are
│ supported.
╵
ERRO[0015] 1 error occurred:
	* exit status 1
```

**Comment by joscabmar** [comment]
I created the following pull request: https://github.com/kuptan/terraform-k8s-sealed-secrets-bootstrap/pull/3

**Comment by IbraheemAlSaady** [comment]
Thanks for fixing this @joscabmar 

I released your changes in [0.3.1](https://github.com/kuptan/terraform-k8s-sealed-secrets-bootstrap/releases/tag/0.3.1)

Closing this issue
